cranelift/codegen/src/isa/unwind.rs

```diff
@@ -27,16 +27,12 @@ pub(crate) mod input {
         SaveRegister {
             offset: CodeOffset,
             reg: Reg,
+            stack_offset: u32,
```

Comment:
For the Windows unwind information, at least, this value has no meaning for integer register saves, only floating point register saves.  I do wonder if we want `Option<u32>` to make that more explicit.

Reply:
On Windows, stack_offset != 0 will be translated to the `UWOP_SAVE_NONVOL` (which will be needed in new backend)

Sequence of StackAlloc {size: WORD_SIZE} + SaveRegister { stack_offset } can be shortened as `UWOP_PUSH_NONVOL`

Reply:
Gotcha, if the new x64 backend will start emitting saves instead of pushes for the integer registers then this makes sense.